include/crocoddyl/core/optctrl/shooting.hpp

```diff
@@ -235,6 +235,12 @@ class ShootingProblemTpl {
    */
   std::size_t get_nthreads() const;
 
+  /**
+   * @brief Print information on the ShootingProblem
```

Comment:
I think to get a proper hyperlink in doxygen we should change ShootingProblem by 'ShootingProblem'